assets/blocks/quiz/quiz.editor.scss

```diff
@@ -7,4 +7,19 @@
 		line-height: 1.25;
 	}
 
+	&__type-selector {
+		&__toolbar {
+			order: -1;
```

Comment:
Why is this needed?

Reply:
This makes sure that it's always the first item in the toolbar, even if there are more toolbar items being added. But it might be a problem for accessibility so I'll remove it for now, hopefully we can control it in JS.